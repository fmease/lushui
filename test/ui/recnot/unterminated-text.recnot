{
    where: "the long
            text starts...
}
