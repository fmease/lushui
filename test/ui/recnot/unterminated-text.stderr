error: unterminated text
  ┌─ ${TEST_FOLDER}/recnot/unterminated-text.recnot:2:12
  │
2 │       where: "the long
  · ╔════════════╝
4 │ ║ }
  │ ╚═╝

error: found end of input but expected ‘,’
  ┌─ ${TEST_FOLDER}/recnot/unterminated-text.recnot:4:3
  │
4 │ }
  │ ⟫⟪

error: aborting due to 2 previous errors

